### DISCUSSION TITLE
Bootstrap Cross-site Scripting Vulnerability

### DISCUSSION THREAD

**Discussion by OMTI-Sunwoo** [twbs/bootstrap#40668] [Q&A]
Hello,

I'm using the kendo-theme-bootstrap 5.3.1 package in my project.
(https://www.npmjs.com/package/@progress/kendo-theme-bootstrap)
The project's dependencies include the bootstrap package, which seems to be where the vulnerability is found.
(https://security.snyk.io/package/npm/bootstrap)

Can I fix this problem?
If not, do you have any plans to fix it?

Thank you for any insight.

**Comment by bardiharborow** [comment]
What version of Bootstrap are you using with kendo-theme-bootstrap 5.3.1? The [package.json](https://github.com/telerik/kendo-themes/blob/v5.3.1/packages/bootstrap/package.json) for that version requires at least v5.1.0 which [has no reported vulnerabilities on Synk](https://security.snyk.io/package/npm/bootstrap).
